src/rpc/net.cpp

```diff
@@ -775,6 +773,54 @@ static UniValue getnodeaddresses(const JSONRPCRequest& request)
     return ret;
 }
 
+static UniValue addpeeraddress(const JSONRPCRequest& request)
```

Comment:
IMO this should be moved to a separate PR, so not reviewing here.

Reply:
I'm keeping it in this PR since it already has reviews and concept ACKs.